### DISCUSSION TITLE
Preferred platform for CWG Meeting

### DISCUSSION THREAD

**Discussion by adamj89** [Green-Software-Foundation/community_wg#65] [General]
Does this community prefer Google Meet or Zoom (or another platform)

**Comment by PindyBhullar** [comment]
Teams or Zoom

**Comment by tmcclell** [comment]
Decision for next meeting.

**Comment by yuxin1234** [comment]
Google meet works for me so far. My colleague had problems with it.

**Comment by alex-kroll** [comment]
Teams or Zoom

**Comment by adamj89** [comment]
Adam to look into CWG account for zoom rather than individual account. Meeting has approved change. No opposition. Motion approved.

**Comment by marcoshidalgonunes** [comment]
For me Google Meet is fine. But it uses lot of space to show the atendants instead to shared screen. Zoom seems to be the best choice.

**Comment by adamj89** [comment]
Issue created to instigate change to Zoom: https://github.com/Green-Software-Foundation/community-wg/issues/70
